src/Middleware/ConcurrencyLimiter/src/QueuePolicies/BasePolicy.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Concurrent;
+using System.Threading.RateLimiting;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.ConcurrencyLimiter;
+
+internal class BasePolicy : IQueuePolicy, IDisposable
+{
+    private readonly System.Threading.RateLimiting.ConcurrencyLimiter _limiter;
```

Comment:
Nit: Do we need the full type name? Same question below in the ctor where `_limiter` is set.

```suggestion
    private readonly ConcurrencyLimiter _limiter;
```

Reply:
It conflicts with the Microsoft.AspNetCore.ConcurrencyLimiter namespace 😢 

ConcurrencyLimiterOptions also directly conflicts between namespaces.